.gitignore

```diff
@@ -3,6 +3,7 @@ packages/react-native/node_modules/
 packages/react-native/lib/
 packages/react-spectrum/node_modules/
 packages/react-spectrum/lib/
+packages/react-components/lib/
```

Comment:
We should rename this to `react-vanilla-components` ? WDYT ?

Reply:
Done!